From 89ea5c62393d550a9afe176c7ad6ac61e6ca590a Mon Sep 17 00:00:00 2001
From: Meng Li <Meng.Li@windriver.com>
Date: Mon, 4 Jul 2022 21:34:30 +0800
Subject: [PATCH] arch: mach-s32: extend the DTB size for BL33

The DTB size should be the same between atf and u-boot, so extend
the DTB size of u-boot to 0x9000, too.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-s32/s32-cc/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-s32/s32-cc/Kconfig b/arch/arm/mach-s32/s32-cc/Kconfig
index a9c68cbe99..636ab4278c 100755
--- a/arch/arm/mach-s32/s32-cc/Kconfig
+++ b/arch/arm/mach-s32/s32-cc/Kconfig
@@ -39,7 +39,7 @@ config SYS_DATA_BASE
 
 config S32CC_MAX_DTB_SIZE
 	hex "Max size reserved for device tree in image layout"
-	default 0x7000
+	default 0x9000
 
 config SYS_MALLOC_F_LEN
 	default 0x2000
-- 
2.36.1

